docs/architecture/adr/0021-console-logging.md

```diff
@@ -0,0 +1,114 @@
+---
+adr: "0021"
+status: In progress
+date: 2023-07-13
+tags: [server]
+---
+
+# 0021 - Logging with the Console
```

Comment:
While console is well termed in the dotnet world due to Windows calling it "Console", it's typically terminal in the linux world. Stdout is a bit more generic.

```suggestion
# 0021 - Logging to stdout
```